src/to-string.cpp

```diff
@@ -235,6 +235,23 @@ const char * get_string( rs2_depth_auto_exposure_mode mode )
 #undef CASE
 }
 
+const char * get_string( rs2_gyro_sensitivity value )
+{
+#define CASE( X ) STRCASE( GYRO_SENSITIVITY, X )
+    switch( value )
+    {
+        CASE( 61_0_MILLI_DEG_SEC )
```

Comment:
Please add a unit test setting the option from python, to validate the option works

Reply:
Next PR :)